samples/ingestion/ingestion-client/StartTranscriptionByTimer/StartTranscriptionHelper.cs

```diff
@@ -269,7 +272,7 @@ private async Task RetryOrFailMessagesAsync(IEnumerable<Message> messages, strin
             }
         }
 
-        private async Task WriteFailedJobLogToStorageAsync(IEnumerable<ServiceBusMessage> serviceBusMessages, string errorMessage, string jobName)
+        private async Task WriteFailedJobLogToStorageAsync(IEnumerable<Connector.ServiceBusMessage> serviceBusMessages, string errorMessage, string jobName)
```

Comment:
Why needed?

Reply:
@JoshLove-msft, There are ServiceBusMessage class in both Connector and Azure.Messaging.ServiceBus.